src/Provider/3dTilesProvider.js

```diff
@@ -101,8 +101,9 @@ export function $3dTilesIndex(tileset, baseURL) {
 
         // getBox only use inverseTileTransform for volume.region so let's not
         // compute the inverse matrix each time
+        // Assumes that node.boundingVolume is defined if node.viewerRequestVolume is undefined
```

Comment:
What is the confidence here ?

Reply:
I think is the specification, and after in the function `getBox` ([here](https://github.com/iTowns/itowns/blob/7d3644e9d9288523d06fbb7fc347c43974bed97a/src/Provider/3dTilesProvider.js#L115))  `node.boundingVolume` is considered defined.

Reply:
Ok